authentication/MultiAuthentication/MultiAuthenticationExtensions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ApplicationParts;
+using MultiAuthentication.Controllers;
+using MultiAuthentication.Handlers;
+using System.Reflection;
+using System.Security.Claims;
+
+namespace MultiAuthentication;
+
+public static class MultiAuthenticationExtensions
```

Comment:
let's leave only the option that we decided to use, and remove the other from code.

it's ok to mention the other option in document as long as it is very very brief